utils/gui/Searchbar.js

```diff
@@ -0,0 +1,252 @@
+import Fetcher from 'Provider/Fetcher';
+import Widget from './Widget';
+
+
+const DEFAULT_OPTIONS = {
+    width: 300,
+    height: 38,
+    position: 'top',
+    maxSuggestionNumber: 10,
+    fontSize: 16,
+};
+
+
+function addActive(htmlElements, index) {
+    if (!htmlElements) { return index; }
+
+    removeAllActives(htmlElements);
+    if (index >= htmlElements.length) { index = 0; }
+    if (index < 0) { index = (htmlElements.length - 1); }
```

Comment:
`.at(-1)`?

Reply:
Cannot use `.at` method with `htmlElements` which is of type `HTMLCollection`